actors/evm/tests/precompile.rs

```diff
@@ -167,7 +175,7 @@ sub
 push1 0x40 # two words
 
 # value
-push1 0x00
+%push({value})
```

Comment:
this should be another parameter to the contract, not written in the bytecode. This way we can return said value and have it part of the assert of `run_test`

Reply:
I agree. But It's just a test thing, so let's fix later.